meinberlin/apps/kiezradar/models.py

```diff
@@ -52,18 +67,18 @@ class SearchProfile(models.Model):
     description = models.TextField(blank=True, null=True)
     disabled = models.BooleanField(default=False)
     notification = models.BooleanField(default=False)
-    status = models.SmallIntegerField(
-        choices=STATUS_CHOICES,
-        default=0,
-        verbose_name=_("Status"),
-    )
```

Comment:
had to change it because in the frontend the status is a multi-select, you can select any combination of choices